docs/knowledge_base/tips/long_callbacks/index.md

```diff
@@ -0,0 +1,73 @@
+# Long Running Callbacks
+
+Tasks in web applications can take a lot of time, which can lead to problems with communication between 
+the server (Python application) and the client (web browser).
+
+![Long Running Callbacks](long_running_callbacks.png){width=100%}
+
+To deal with this issue, Taipy has a helpful feature called "long-running callbacks" to address this problem. 
+These callbacks allow the server to do demanding tasks in the background while still making sure 
+the user interface stays quick to respond. 
+This article talks about the idea of long-running callbacks in Taipy, gives examples of how to use them, 
+and shows how they make the user experience better.
+
+## Example 1: Executing a Heavy Task in the Background
```

Comment:
If we remove the H1 title, we can change the title level accordingly.
```suggestion
# Example 1: Executing a Heavy Task in the Background
```

Reply:
May be not though.
MkDocs adds a H1-like thing, and using a H2 section makes the section title smaller, which is probably what we want.

Reply:
Oh... I created dozens of comments in other PRs about that.
🤣🤣🤣🤣
 
We need to make a global decision, I guess.